src/js/services/dialer.js

```diff
@@ -34,6 +35,44 @@ export default App.extend({
   call(number, action) {
     this._call(number, action);
   },
+  five9CallNumber(values) {
+    if (!values) {
```

Comment:
If `null` is passed like: `Radio.request('dialer', 'five9CallNumber', null);`, empty the patients collection. Removing any patient dashboard buttons visible in the UI. As it's configured now, that would be on call end.